fern/docs/pages/manual/settings.mdx

```diff
@@ -77,4 +77,19 @@ Missing variables with no default will produce an error.
 ```yaml
 server:
   port: ${PORT:8001}
+```
+
+## LLM config options
```

Comment:
I think this fits better in 

"Customizing low level parameters" within Installation

or in 

"Advanced setup" > "LLM Backends"

Reply:
Good idea - i've moved it to the former section